src/Tool-Workspace/Workspace.class.st

```diff
@@ -146,6 +146,20 @@ Workspace >> browseBindings [
 	bindings inspect
 ]
 
+{ #category : #necompletion }
```

Comment:
Is it really right to change the package containing this method? It seems to add a new dependency to Tool-Workspace.